packages/perseus-editor/src/widgets/number-line-editor.tsx

```diff
@@ -389,9 +374,9 @@ class NumberLineEditor extends React.Component<Props> {
                     {!this.props.static && (
                         <Checkbox
                             label="Show tooltips"
-                            checked={this.props.showTooltips}
+                            checked={this.props.showTooltip}
                             onChange={(value) => {
-                                this.props.onChange({showTooltips: value});
+                                this.props.onChange({showTooltip: value});
```

Comment:
This makes me nervous.

Reply:
Rightly. :)  It should be singular as noted above. :)